app/assets/stylesheets/style.css.scss

```diff
@@ -181,6 +184,10 @@ a {
   }
 }
 
+.title {
+  padding-top: 100px;
```

Comment:
I have added it to the padding to the `#main-content` and I tried making the menu toggle-able, but didn't get it working the way I wanted, because I am not that good with CSS. I also tried with the help of the link you sent me and used that as a base, but still didn't get it working.